src/main/java/seedu/address/logic/commands/ClearLogCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.LogBook;
+import seedu.address.model.Model;
+
+/**
+ * Clears the address book.
+ */
+public class ClearLogCommand extends Command {
+
+    public static final String COMMAND_WORD = "clog";
+
+    public static final String COMMAND_WORD_ALIAS = "cl";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "or" + COMMAND_WORD_ALIAS
+            + ": Clears the entire logger tab.\n"
+            + "Example: " + COMMAND_WORD + "or" + COMMAND_WORD_ALIAS;
```

Comment:
Same thoughts as the one `AppendLogCommand`, can make it 2 examples.
```suggestion
            + "Example 1: " + COMMAND_WORD + "\n"
            + "Example 2: " + COMMAND_WORD_ALIAS;
```